src/test/java/hudson/plugins/sonar/MsBuildSQRunnerBeginTest.java

```diff
@@ -110,6 +112,19 @@ public void testNoMsBuildInst() throws Exception {
     assertLogContains("No SonarScanner for MSBuild installation found", r);
   }
 
+  @Test
+  public void testDotNetCoreScanner() throws Exception {
+    configureDefaultSonar();
+    configureMsBuildScanner(false);
+
+    FreeStyleProject proj = createFreeStyleProjectWithMSBuild("default", "default");
+    MsBuildSQRunnerInstallation.setTestExeName("Foo.dll");
+    Run<?, ?> r = build(proj, Result.FAILURE);
+
+    String log = JenkinsRule.getLog(r);
+    assertThat(log).containsPattern("dotnet .*Foo.dll begin /k:key /n:name /v:1.0");
```

Comment:
Use `containsPattern` because I couldn't find a way to get the full path which is the prefix of `Foo.dll`.